### PR TITLE
Image alt tag accessibility fix

### LINKED ISSUE #1128: Unable to clear alt text, video and image elements
## Bug Description
I am unable to clear the alt / title accessibility fields on the video element. When I empty the field, either by using the delete key or cross, it refills with default value again. 

## Expected Behaviour
User should be able to clear field. 

## Steps to Reproduce

See video. 

## Screenshots

https://youtu.be/DEq9MiOyjI4

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by diegovar**
Similar to https://github.com/google/web-stories-wp/pull/1340, fixes #1128

### CODE REVIEW

**Comment by spacedmonkey** [comment]
Seems like this bug is #1128

Any chance you could also fix the other fields I reference in the ticket?

File: assets/src/edit-story/components/panels/videoAccessibility.js [hunk 1]
```diff
@@ -41,9 +41,21 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const poster = getCommonValue(selectedElements, 'poster') || resource.poster;
-  const title = getCommonValue(selectedElements, 'title') || resource.title;
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
```
Comment (spacedmonkey) [inline assets/src/edit-story/components/panels/videoAccessibility.js:45]: Why is this needed? Doesn't `getCommonValue` already have a param default value? Why can't we use that?

File: assets/src/edit-story/components/panels/imageAccessibility.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ function ImageAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+  const altValue = getCommonValue(selectedElements, 'alt');
```
Comment (dvoytenko) [inline assets/src/edit-story/components/panels/imageAccessibility.js:43]: Ditto: please check the 3rd arg.

**Review by dvoytenko — CHANGES_REQUESTED** [review]

**Comment by diegovar** [comment]
Closing as I'll move the changes to https://github.com/google/web-stories-wp/pull/1340

**Pull request closed without merging**